libnavui-androidauto/src/main/java/com/mapbox/androidauto/navigation/ActiveGuidanceScreen.kt

```diff
@@ -73,24 +70,15 @@ internal class ActiveGuidanceScreen constructor(
         })
     }
 
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
     override fun onGetTemplate(): Template {
         logAndroidAuto("ActiveGuidanceScreen onGetTemplate")
-        val actionStrip = ActionStrip.Builder().apply {
-            actionProviders.forEach {
-                this.addAction(it.getAction(this@ActiveGuidanceScreen))
-            }
-            this.addAction(
-                Action.Builder()
-                    .setTitle(carContext.getString(R.string.car_action_navigation_stop_button))
-                    .setOnClickListener {
-                        carArrivalTrigger.triggerArrival()
-                    }.build()
-            )
-        }.build()
-
         return NavigationTemplate.Builder()
             .setBackgroundColor(CarColor.PRIMARY)
-            .setActionStrip(actionStrip)
+            .setActionStrip(
+                mapboxCarContext.options.actionStripProvider
+                    .getActionStrip(this, MapboxScreen.ACTIVE_GUIDANCE)
```

Comment:
Why don't we directly call `getActiveGuidance` here? We can make it and other functions public and remove `getActionStrip`.

Reply:
I've made it so there is a top level `getActionStrip` function which is the only public accessor. This way, someone downstream can override all of the action buttons with a single function.

They also have the option to only override the `protected fun getActiveGuidance`